internals/generators/container/stateless.js.hbs

```diff
@@ -46,7 +55,7 @@ function {{ properCase name }}() {
 
 {{#if wantActionsAndReducer}}
 const mapStateToProps = createStructuredSelector({
-  {{name}}: makeSelect{{properCase name}}(),
+  {{ lowerCase name }}: makeSelect{{properCase name}}(),
```

Comment:
The generator was replaced here in 3.4, the latest version - https://github.com/react-boilerplate/react-boilerplate/pull/1331/commits/4a0265b3d4b9f65154d2627a82b2df3f3bdbb1e6

IMO, it seems like a bad default - prefixing the props of a container with the container's name is redundant - it's only slightly better when lowercasing it ;) And I feel complex cases should still be handled on a case-by-case basis in selectors.js since that file is specifically for selectors, sharing selector code between selectors.js and index.js is quite strange to me.